extensions/mssql/src/util/dataCache.ts

```diff
@@ -0,0 +1,40 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+export class DataCache {
+
+	millisecondsToLive: number;
+	getValueFunction: (...args: any[]) => any;
```

Comment:
Why are you using any? You can make generic classes in TS

Reply:
Oh really? I am unfamiliar with that syntax in TS. I will look it up.